sycl/test-e2e/Regression/check_vector_of_opencl_event.cpp

```diff
@@ -1,5 +1,5 @@
 // REQUIRES: opencl
-// RUN: %{build} -o %t.out
+// RUN: %{build} -std=c++20 -o %t.out
```

Comment:
Why do we need this?

Reply:
This warning is generated without it:
``` 
# .---command stderr------------
# | /localdisk2/dgarciao/llvm/sycl/test-e2e/Regression/check_vector_of_opencl_event.cpp:21:38: warning: use of function template name with no prior declaration in function call with explicit template arguments is a C++20 extension [-Wc++20-extensions]
# |    21 |   std::vector<cl_event> ClEventVec = get_native<sycl::backend::opencl>(event);
# |       |                                      ^
# | 1 warning generated.
# | /localdisk2/dgarciao/llvm/sycl/test-e2e/Regression/check_vector_of_opencl_event.cpp:21:38: warning: use of function template name with no prior declaration in function call with explicit template arguments is a C++20 extension [-Wc++20-extensions]
# |    21 |   std::vector<cl_event> ClEventVec = get_native<sycl::backend::opencl>(event);
# |       |                                      ^
# | 1 warning generated.
# \`-----------------------------
```

Reply:
Folks this is the wrong way to fix this warning. The test is not supposed to use C++20 features.

Please replace `get_native<sycl::backend::opencl>(event)` to `sycl::get_native<sycl::backend::opencl>(event)` at line 20.